libs/MVS/SceneTexture.cpp

```diff
@@ -1677,7 +1677,8 @@ void MeshTexture::PoissonBlending(const Image32F3& src, Image32F3& dst, const Im
 			const MatIdx idx(indices(i));
 			ASSERT(idx != -1);
 			coeffA.AddConstruct(idx, idx, 1.f);
-			coeffB[idx] = Color(dst(i));
+			Color c = dst(i);
```

Comment:
There should be a nicer solution to this. Could you please try something different to fix it (without creating a temporary variable)?